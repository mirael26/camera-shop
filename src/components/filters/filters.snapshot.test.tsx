import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import Filters from './filters';

jest.mock('./category-filter/category-filter', () => 'CategoryFilter');
jest.mock('./level-filter/level-filter', () => 'LevelFilter');
jest.mock('./price-filter/price-filter', () => 'PriceFilter');
jest.mock('./type-filter/type-filter', () => 'TypeFilter');

test('Filters render correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<Filters/>);
  expect(asFragment()).toMatchSnapshot();
});
