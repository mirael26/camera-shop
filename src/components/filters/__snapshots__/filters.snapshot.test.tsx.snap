// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filters render correctly 1`] = `
<DocumentFragment>
  <div
    class="catalog-filter"
  >
    <form
      action="#"
    >
      <h2
        class="visually-hidden"
      >
        Фильтр
      </h2>
      <pricefilter />
      <categoryfilter />
      <typefilter />
      <levelfilter />
      <button
        class="btn catalog-filter__reset-btn"
        type="reset"
      >
        Сбросить фильтры
      </button>
    </form>
  </div>
</DocumentFragment>
`;
